index.js

```diff
@@ -0,0 +1,358 @@
+'use strict';
+
+const Scm = require('screwdriver-scm-base');
+const async = require('async');
+const hoek = require('hoek');
+
+class ScmRouter extends Scm {
+    /**
+     * Constructs a router for different scm strategies
+     * @method constructor
+     * @param  {Object}         config                             Object with scms and ecosystem
+     * @param  {Object}         [config.ecosystem]                 Optional object with ecosystem values
```

Comment:
We don't need `ecosystem` for scm plugins. Can we remove all the related code?
Same for the API PR https://github.com/screwdriver-cd/screwdriver/pull/648/files